No necesitar hacer click y reescribir

jkcgs opened the pull request:
El script se ejecuta cada medio segundo dado a que los artículos se cargan de forma asincrónica. Una vez que no queden más artículos internacionales, no se sigue ejecutando. Además, reescribir el pequeño script.

jkcgs committed:
No necesitar hacer click

ivanMSC commented:
Gracias @jkcgs ! Lo probé y encontré que se bugea de repente. Esto pasa en https://listado.mercadolibre.cl/canon#D[A:canon]

![image](https://user-images.githubusercontent.com/35350256/95528385-57fba080-09ae-11eb-9a95-bf83dbdc2d51.png)

Mientras que con la version publicada se obtiene lo siguiente:
![image](https://user-images.githubusercontent.com/35350256/95528422-7497d880-09ae-11eb-9a7b-803e0d538620.png)

Desconozco a qué se pueda deber. Si puedes darle una vuelta, sería bacan para tener la extensión funcionando de forma autonóma.

jkcgs committed:
No eliminar, solo ocultar los resultados

Además, solo ejecutar si es una página de resultados

jkcgs commented:
Otro intento con otro método un poco menos simple

jkcgs committed:
Ordenar y volver a método de selección anterior

Apuntar nuevamente a los flairs (e.g., "Internacional"), en vez de a cada item `<li>`.

jkcgs committed:
Simplificar

Quitar referencia a `window` y auto-llamar función.

jkcgs commented:
Volví a reescribir el código de forma más """elegante""" y al parecer funciona más rápido.

jkcgs committed:
Mover hartas cosas

jkcgs committed:
Links raw no funcionan en readme

jkcgs committed:
Workaround para link de userscript

ivanMSC commented:
Funciona perfecto
